repos/system_upgrade/el8toel9/actors/addarmbootloaderworkaround/tests/test_addarmbootloaderworkaround.py

```diff
@@ -0,0 +1,159 @@
+import os
+
+import pytest
+
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.actor import addupgradebootloader
+from leapp.libraries.common.testutils import CurrentActorMocked, logger_mocked, make_OSError
+from leapp.libraries.stdlib import api
+from leapp.models import TargetUserSpaceInfo
+
+
+class IsolatedActionsMocked(object):
```

Comment:
If I am not missing something this is unused.

Reply:
It was a leftover from some test that were deleted at the time. I've made it relevant again with new tests.